src/util/panic_hook.rs

```diff
@@ -65,7 +68,28 @@ fn track_hook(p: &PanicInfo) {
 pub fn set_exit_hook() {
     let orig_hook = panic::take_hook();
     panic::set_hook(box move |info: &PanicInfo| {
-        orig_hook(info);
+        if log_enabled!(LogLevel::Error) {
+            let msg = match info.payload().downcast_ref::<&'static str>() {
```

Comment:
The log output file is global and can't be changed outside log crate after being initialized.